src/Repository/LogActionItemRepository.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace App\Repository;
+
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Persistence\ManagerRegistry;
+use Doctrine\ORM\Query;
+
+use App\Entity\Dataset;
+use App\Entity\LogActionItem;
+use App\Util\FundingOrgFilter;
+
+/**
+ * LogActionItem Entity Repository class.
+ */
+class LogActionItemRepository extends ServiceEntityRepository
+{
+    /**
+     * Utility to filter by funding organization.
+     *
+     * @var FundingOrgFilter
+     */
+    private $fundingOrgFilter;
+    
+    /**
+     * Constructor.
+     *
+     * @param ManagerRegistry  $registry         The Registry Manager.
+     * @param FundingOrgFilter $fundingOrgFilter Utility to filter by funding organization.
+     */
+    public function __construct(ManagerRegistry $registry, FundingOrgFilter $fundingOrgFilter)
+    {
+        parent::__construct($registry, LogActionItem::class);
+        
+        $this->fundingOrgFilter = $fundingOrgFilter;
+    }
+    
+    /**
+     * Sum of all dataset file sizes.
+     *
+     * @return integer Size of data in bytes.
+     */
+    public function countDownloads() : int
+    {
+        $qb = $this->createQueryBuilder('log')
+            ->select('COUNT(log.id)')
+            ->where('log.subjectEntityName = ?1')
+            ->andWhere('log.actionName = ?2')
+            ->setParameter(1, 'Pelagos\Entity\Dataset')
+            ->setParameter(2, 'File Download');
+
+        if ($this->fundingOrgFilter->isActive()) {
+            $researchGroupIds = $this->fundingOrgFilter->getResearchGroupsIdArray();
```

Comment:
So this could mean that, a GRP dataset that is downloaded from the GRIIDC site would also show up on GRP stats page

Reply:
That's true, not sure what the desired behavior is. We don't track from which "site" it was downloaded.

Reply:
That is ok, the GRP dataset is the same as the GRIIDC dataset, we just want the total counts.  Same with views if we implement them.  I could see adding something to the download report that indicates which data landing page it was downloaded from but that doesn't affect this.

Reply:
> That is ok, the GRP dataset is the same as the GRIIDC dataset, we just want the total counts. Same with views if we implement them. I could see adding something to the download report that indicates which data landing page it was downloaded from but that doesn't affect this.

Who this, new phone!?

Reply:
I don't know why I have multiple github accounts but you can be sure I will always be logged into the wrong one

Reply:
If we ever wanted counts per "site" in a report, we'd need to add it to the log action item payload.